Sources/Player/PlaybackEngine/Internal/AudioSessionMonitors/AudioSessionRouteChangeMonitor.swift

```diff
@@ -35,13 +35,15 @@
 			      let reasonValue = userInfo[AVAudioSessionRouteChangeReasonKey] as? UInt,
 			      let reason = AVAudioSession.RouteChangeReason(rawValue: reasonValue)
 			else {
+				PlayerWorld.logger?.log(loggable: PlayerLoggable.changeMonitorHandleNotificationWithoutRequiredData)
 				return
 			}
 
 			switch reason {
 			case .oldDeviceUnavailable:
 				playerEngine?.pause()
 			default: ()
+				PlayerWorld.logger?.log(loggable: PlayerLoggable.changeMonitorHandleNotificationDefaultReason(reason: String(describing: reason)))
```

Comment:
Why do we need to log this if we don't want to do anything in the other route change reasons?

Reply:
No. the only issue could be that it is triggered a lot, but I don't think so.